### DISCUSSION TITLE
seek help for me

### DISCUSSION THREAD

**Discussion by hanyi0914** [STranslate/STranslate#205] [Q&A] [answered]
Hello there! I’m really thrilled to be using this product. However, I’ve encountered a bit of an issue that I hope you can help with. I can install and use it just fine on Windows, but I’m having trouble getting it to work on my Mac. I’d love to see if your team can plan for a similar version that’s compatible with Mac computers. Thanks for your attention!

**Comment by ZGGSONG** [comment, marked as answer]
Due to technical limitations, the technology used in the software development is the WPF framework, which is a UI framework that can only run on the Windows platform and cannot cross platforms, so it is not compatible
